README.md

```diff
@@ -442,7 +473,9 @@ new client.Histogram({
 The content-type prometheus expects is also exported as a constant, both on the
 `register` and from the main file of this project, called `contentType`.
 
-## Garbage Collection
+### Garbage Collection Metrics
 
-To avoid dependencies in this module, GC stats are kept outside of it. If you
-want GC stats, you can use https://github.com/SimenB/node-prometheus-gc-stats
+To avoid native dependencies in this module, GC statistics for bytes reclaimed
```

Comment:
Isn't the data provided by that module now provided using the builtin APIs?

Reply:
Everything except "bytes reclaimed" is now via builtin APIs. You might actually know the answer to this, or know who to ask... `gc-stats` just calls `GetHeapStatistics` in the GC prologue and epilogue and diffs them to calculate "bytes reclaimed". But since v8's GC is concurrent, I don't think that's an accurate method. Do you know if that's true? If so, we should remove this section of the readme.

---

Thanks for the review, agree with all the other changes.

Reply:
It might be accurate, but I'm not sure its useful, I suspect it might have been emitted just because it seemed possible. If many bytes are reclaimed, is that good? If only a few are reclaimed, is that good? The total usage and how it varies with time strike me as the useful metrics, and also allow diff scrape-to-scrape.